fdbcli/fdbcli.actor.cpp

```diff
@@ -525,6 +525,11 @@ void initHelp() {
 	helpMap["getversion"] =
 	    CommandHelp("getversion", "Fetch the current read version",
 	                "Displays the current read version of the database or currently running transaction.");
+	helpMap["advanceversion"] = CommandHelp(
```

Comment:
If the cluster has already been recovered for some time and its current version V1 is much larger than the configured version V_c, will it have no _immediate_ effect on the cluster except setting a high `minRequiredCommitVersionKey` value?

What will happen if the cluster has another recovery between V_c and V1?

Reply:
If v1 > vc, this command will have no effect. It just ensures that once the `advanceversion` command have finished, all future read versions given out will be `>= vc`.